<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Area</class>
 <widget class="QWidget" name="Area">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Area</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>current area</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="languageframe_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>960</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="countrylabel">
        <property name="minimumSize">
         <size>
          <width>209</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>209</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Area showing time currency format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="countrycomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>291</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="title2Label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Regional format data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="formframe">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>118</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>960</width>
       <height>118</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="calendarlabel">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>calendar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Lunarcalendar">
          <property name="text">
           <string>lunar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="weeklabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>First day of the week</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firstDayLabel">
          <property name="text">
           <string>monday</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="datelabel">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="datelabelshow">
          <property name="text">
           <string>2019/12/17</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="timelabel">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timelabelshow">
          <property name="text">
           <string>9:52</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="bottomMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QPushButton" name="chgformButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>change format of data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="title3Label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="languageframe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>960</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>16</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="languagelabel">
          <property name="minimumSize">
           <size>
            <width>209</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>209</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="langcomboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="addlanframe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>960</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="addlanBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="addlanlabel">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>538</width>
            <height>39</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
